chall03/alcohen.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var gTimeStart = time.Now()
+
+func elapsedTime() time.Duration {
+	return time.Since(gTimeStart) / 1e6
+}
+
+func makeGetRequest(url string) string {
```

Comment:
http.Get() is already a net/http function that actually do that. I would suggest not to create a function for that, so if later you are asked to use a response Header or the response status code, you will not have to find a way to pass the parameters back to your main function.